core/src/main/java/io/undertow/UndertowOptions.java

```diff
@@ -325,6 +325,13 @@ public class UndertowOptions {
      */
     public static final Option<Boolean> SSL_USER_CIPHER_SUITES_ORDER = Option.simple(UndertowOptions.class, "SSL_USER_CIPHER_SUITES_ORDER", Boolean.class);
 
+    /**
+     * This option forces {@link io.undertow.protocols.ssl.UndertowXnioSsl} to use a specific
+     * name as the {@link javax.net.ssl.SNIHostName} for a client connection. If the option is
+     * not set, the hostname in the URL, if available (i.e. not using IP), is used. The option
```

Comment:
"If the option is not set, the hostname in the URL, if available (i.e. not using IP), is used".
It would look clearer if this were reorded as in:
"If the option is not set, the hosname in the URL is used, if available (i.e., not using IP)."
also, I would replace the second if by a when just to prevent using the same proposition twice in the same statement and take away the "in the":
"If the option is not set, the URL hostname is used when available (for URLs that do not use the IP address)

Reply:
Done!